doc/modules/outlier_detection.rst

```diff
@@ -302,3 +318,25 @@ multiple modes and :class:`ensemble.IsolationForest` and
      an outlier detection method), the :class:`ensemble.IsolationForest`,
      the :class:`neighbors.LocalOutlierFactor`
      and a covariance-based outlier detection :class:`covariance.EllipticEnvelope`.
+
+One-class SVM versus SVDD-L1
```

Comment:
`One-Class` is used in the rest of the docs instead of `One-class`, perhaps you should change to maintain consistency?